addons/binding/org.openhab.binding.pjlinkdevice/README.md

```diff
@@ -0,0 +1,50 @@
+# PJLink Binding
+
+This binding allows you to control devices with [PJLink](https://pjlink.jbmia.or.jp/english/) support. PJLink is a protocol that was developed to standardize IP control of digital projectors, but can also be used by other device types.
+
+Aspects that can be controlled via PJLink are power on/off, input source selection and volume/mute setting.
+
+## Supported Things
+
+Supported are devices which support the PJLink protocol (class 1). 
+
+Limitations at this point:
+
+- only IPv4 connections are supported
+- only PJLink class 1 commands are supported
+
+The binding is tested with the PJLink device test tool (PJLinkTEST4CNT) and an Acer VL7680.
+
+## Discovery
+
+Autodiscovery is checking all IP addressess of all class C IPv4 subnets connected to the openHAB system for devices which respond PJLink compliant on the PJLink standard port 4352.
+
+## Thing Configuration
+
+The *pjLinkDevice* thing type has the following parameters:
+
+- *ipAddress*: the IPv4 address of the device
```

Comment:
Use table notation for configuration and channels. It's more readable:
```
| Parameter | Description                    |
|-----------|--------------------------------|
| ipAddress | the IPv4 address of the device |
```
Make sure to the bars are all aligned vertically in the raw text.

Reply:
done